### ISSUE TITLE
Cannot use object of type CallFeedItem as array

### ISSUE THREAD

**Issue by cacapan** [googleads/googleads-php-lib#192]
Hi,
I am using BatchJobUtils to create ad group extension setting.
The add operations works fine and got response correctly.
`<mutateResponse xmlns="https://adwords.google.com/api/adwords/cm/v201609">
   ... ...
   <rval>
      <result>
         <AdGroupExtensionSetting>
            <extensionType>CALL</extensionType>
            <extensionSetting>
               <extensions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="CallFeedItem">
                   ... ...
               </extensions>
               <platformRestrictions>NONE</platformRestrictions>
             </extensionSetting>
         </AdGroupExtensionSetting>
      </result>
      <index>55</index>
   </rval>
   ... ...
</mutateResponse>`
But I got this error converting xml to object

`Cannot use object of type CallFeedItem as array`
in PATH/TO/UTIL/XmlDeserializer.php on line 105

Thanks in advance.
Cheers.

**Comment by fiboknacky** [comment]
Hello @cacapan,

Could you please provide your the library version you're using and your batch job ID as well?
Thanks in advance!

Best,
Knack

**Comment by cacapan** [comment]
Hi @fiboknacky 

lib version is v14.0.0 and batch job Id is 422840004
Thanks for your help.

Cheers.

**Comment by fiboknacky** [comment]
Hello @cacapan 

Thanks for your information. :)
I'll have a look at this.

Best,
Knack

**Comment by fiboknacky** [comment]
Hello @cacapan

We're working on the fix and will push the fix later.
If you wish to fix this by yourself first, please go to [`BatchJobClasses`](https://github.com/googleads/googleads-php-lib/blob/deprecated/src/Google/Api/Ads/AdWords/Util/v201609/BatchJobClasses.php) and add  `$AdGroupExtensionSetting` as a member variable of the class [`Operand`](https://github.com/googleads/googleads-php-lib/blob/deprecated/src/Google/Api/Ads/AdWords/Util/v201609/BatchJobClasses.php#L116) like this:
```php
    /**
     * @var AdGroupExtensionSetting
     */
    public $AdGroupExtensionSetting;
```

Hope this help.

Best,
Knack

**Comment by vtsao** [comment]
Fixed in [16.0.0](https://github.com/googleads/googleads-php-lib/releases/tag/16.0.0).
